src/main/java/org/cascadebot/cascadebot/commands/fun/RandomCommand.java

```diff
@@ -49,4 +53,9 @@ public CascadePermission getPermission() {
         return CascadePermission.of("random", true);
```

Comment:
Why is this in onCommand?

Is it even in onCommand? All the file joins in the GitHub file viewer are throwing me off here :(

Reply:
It's not in onCommand ;)